### PR TITLE
Adjust severity filtering rules

### LINKED ISSUE #126: Adjust severity filtering rules
Depends on #101 

When you run console.log("foo"), you get three lines

``` js
console.log("foo") //console command
foo                // console log
undefined          // evaluation result
```

If the log filter is turned off, the console log should not show up, but the evaluation result and console command should.

### PR DESCRIPTION
**Pull request by linclark**
Fixes #126 
## Testing instructions
1. Add the following messages
   
   ```
   console.log("asdf")
   console.count("bar")
   ```
2. Toggle the Log filter on and off. You should only see the log message go away (before, the console command and evaluation result would as well)
3. Toggle the Debug filter. You should see the count message go away.

### CODE REVIEW

**Comment by linclark** [comment]
This depends on #179

**Comment by linclark** [comment]
@bgrins this is ready for review

File: devtools/client/webconsole/new-console-output/reducers/filters.js [hunk 1]
```diff
@@ -9,11 +9,12 @@ const Immutable = require("devtools/client/shared/vendor/immutable");
...
 
 const FilterState = Immutable.Record({
+  debug: true,
   error: true,
-  warn: true,
   info: true,
   log: true,
-  text: ""
+  text: "",
+  warn: true
```
Comment (bgrins) [inline devtools/client/webconsole/new-console-output/reducers/filters.js:17]: Nit: please include trailing comma after last property

**Comment by bgrins** [comment]
Just a nit, looks good to m

**Pull request merged**